### PR TITLE
Feature/layout review 

### PR DESCRIPTION
**Pull request by Sathvik-Chowdary-Veerapaneni**
- This feature branch consists of design layout page code
- Modifications to design layout 
-  Created buttons to channels and messages columns
-  Alignments to box fields  
- Created settings and username change buttons

### CODE REVIEW

File: flask-discord/flaskd/templates/layout.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+<title> MONEY HUNTERS </title>
+
+</head>
+<body style="background-color: #565454;"> 
+<center>
+<table>
+
+<tr>
+
+<td colspan="3" bgcolor="gray">
```
Comment (hemanth5143) [inline flask-discord/flaskd/templates/layout.html:19]: Background color contrast as grey looking decent.

**Review by hemanth5143 — APPROVED** [review]
looking simple UI and good work guys

**Review by spyralspy01 — APPROVED** [review]
simple layout. works as intended.

**Comment by Sathvik-Chowdary-Veerapaneni** [comment]
Thank you Guys !..

**Pull request merged**